### PR TITLE
Justin Arata: Stretch

### PR DESCRIPTION
**Pull request by jarata**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -40,3 +40,24 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let nav = document.querySelectorAll('a');
```
Comment (szincone) [inline js/index.js]: Good job on using your querySelectorAll to get all of your a tags on the page.

File: js/index.js [hunk 2]
```diff
@@ -40,3 +40,24 @@ const siteContent = {
...
+nav[1].textContent = 'Product';
+nav[1].style.color = 'grey'
+nav[2].textContent = 'Vision';
+nav[2].style.color = 'grey'
+nav[3].textContent = 'Features';
+nav[3].style.color = 'grey'
+nav[4].textContent = 'About';
+nav[4].style.color = 'grey'
+nav[5].textContent = 'Contact';
+nav[5].style.color = 'grey'
```
Comment (szincone) [inline js/index.js]: Could possibly use a for loop or forEach loop to style all the nav items colors too. `nav.forEach(item => item.style.color = 'grey')`

File: js/index.js [hunk 3]
```diff
@@ -38,26 +38,60 @@ const siteContent = {
...
+let ctaH1 = document.querySelector('h1').innerHTML = siteContent["cta"]["h1"];
+// ctaH1.textContent = 'DOM Is Awesome';
+let ctaButton = document.querySelector('button').innerHTML = siteContent["cta"]["button"];
+// ctaButton.textContent = 'Get Started';
 let ctaLogo = document.getElementById("cta-img");
-ctaLogo.setAttribute('src', siteContent[".cta"]["img-src"])
+ctaLogo.setAttribute('src', siteContent["cta"]["img-src"]);
+
+let textContentDiv = document.querySelectorAll(".text-content");
+textContentDiv[0].querySelector('h4').innerHTML = siteContent['main-content']['features-h4'];
```
Comment (ashwins93) [inline js/index.js]: Prefer using `textContent` over `innerHTML` wherever possible for security reasons

File: js/index.js [hunk 4]
```diff
@@ -43,19 +43,22 @@ let container = document.querySelectorAll('.container')
...
-nav[2].style.color = 'grey';
-nav[3].textContent = 'Features';
-nav[3].style.color = 'grey';
-nav[4].textContent = 'About';
-nav[4].style.color = 'grey';
-nav[5].textContent = 'Contact';
-nav[5].style.color = 'grey';
+let nav = document.querySelectorAll('a')
+for(i = 0; i < nav.length; i++) {
+  nav[i].textContent = siteContent['nav'][`nav-item-${i+1}`]
```
Comment (ashwins93) [inline js/index.js]: That's a very clean refactor. Nice.

File: js/index.js [hunk 5]
```diff
@@ -94,4 +94,19 @@ contactP[1].innerHTML = siteContent["contact"]["phone"];
...
+
+appendChild.href = '#';
+appendChild.textContent = "AppendChild!"
+
+let navBar = document.querySelector('nav');
+
+navBar.prepend(prepend);
+navBar.appendChild(appendChild);
+
+nav.forEach((e) => e.style.color = 'green');
```
Comment (ashwins93) [inline js/index.js]: You need to re-select `nav` in order to effect the changes for the newly added items to the nav.

**Pull request closed without merging**